example/run_demo.py

```diff
@@ -34,10 +35,10 @@
 SAMPLE_APP_DIR = Path("coolstore")
 ANALYSIS_BUNDLE_PATH = Path(".", "analysis", "bundle.jar")
 ANALYSIS_LSP_PATH = Path(".", "analysis", "jdtls", "bin", "jdtls")
-ANALYSIS_RPC_PATH = Path(".", "analysis", "kai-analyzer-rpc")
+ANALYSIS_RPC_PATH = Path(".", "analysis",  f"kai-analyzer-rpc{'.exe' if platform.system().lower() == 'windows' else ''}")
```

Comment:
this makes sense, but may be nice to be in a helper function something like get_binary_path that will append `.exe` if it does not currently exist in case we ever add arguments to the run_demo to pass these in. 

Thoughts?

Reply:
Yeah that makes totally sense, take look now, I ddn't know where exactly to place that function